books/free-programming-books-langs.md

```diff
@@ -2167,6 +2167,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 
 * [Developing Time-Oriented Database Applications in SQL](https://www2.cs.arizona.edu/~rts/tdbbook.pdf) - Richard T. Snodgrass (PDF)
 * [Introduction to SQL](https://github.com/bobbyiliev/introduction-to-sql) - Bobby Iliev (Markdown, PDF)
+* [SQLite Tutorial](https://www.tutorialspoint.com/sqlite)
```

Comment:
It's rare that linter warns about L1209 but maybe referes to place it below `SQL Queries Succinctly` (at L1173)